legend-engine-test-runner-service/src/main/java/org/finos/legend/engine/testable/service/connection/TestConnectionBuilder.java

```diff
@@ -65,7 +65,13 @@ public Pair<Connection, List<Closeable>> visit(Connection connection)
                 .select(Optional::isPresent)
                 .collect(Optional::get)
                 .getFirstOptional()
-                .orElseThrow(() -> new UnsupportedOperationException("Unsupported embedded data type '" + embeddedData.getClass().getSimpleName() + "\" with connection type '" + connection.getClass().getSimpleName() + '"'));
+                .orElseThrow(() ->
+                {
```

Comment:
Can we please add a test ? We already have a test class - TestTestConnectionBuilder

Reply:
Done